data/iso_iec_dir_1_iso_sup_ed-11-2020.yml

```diff
@@ -0,0 +1,59 @@
+title:
+- type: title-intro
+  content: ISO/IEC Directives
+  language: en
+  script: Latn
+  format: text/plain
+- type: title-main
+  content: Consolidated ISO Supplement
+  language: en
+  script: Latn
+- type: title-part
+  content: Procedures specific to ISO
+  language: en
+  script: Latn
+- type: title-intro
+  content: Directives ISO/IEC
+  language: fr
+  script: Latn
+  format: text/plain
+- type: title-main
+  content: Supplément ISO consolidé
+  language: fr
+  script: Latn
+- type: title-part
+  content: Procédures spécifiques à l'ISO
+  language: fr
+  script: Latn
+
+part: "1"
+
+link:
+- type: src
+  content: https://fdocuments.in/document/isoiec-directives-part-1-consolidated-iso-supplement-and-iso-proceduresiso.html
+- type: obp
+  content: https://fdocuments.in/embed/v1/isoiec-directives-part-1-consolidated-iso-supplement-and-iso-proceduresiso.html
```

Comment:
The OBP is the ISO Online Browsing Platform. The links start with https://www.iso.org/obp. There is no need to fill this in if it is not from the OBP.